apps/homescreen/manifest.webapp

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "New Home Screen",
+  "name": "Default Home Screen",
```

Comment:
In order to make this string localizable, we should do the same as in https://github.com/mozilla-b2g/gaia/blob/master/apps/verticalhome/manifest.webapp#L21